src/ERC721Drop.sol

```diff
@@ -1267,6 +1272,38 @@ contract ERC721Drop is
         emit MintFeePayout(zoraFee, ZORA_MINT_FEE_RECIPIENT, success);
     }
 
+    function _requireCanMintQuantity(uint256 quantity) internal view {
+        if (quantity + _totalMinted() > config.editionSize) {
+            revert Mint_SoldOut();
+        }
+    }
+
+    function _handleSupplyRoyalty(uint256 mintAmount) internal returns (uint256 totalRoyaltyMints) {
+        if (royaltyMintSchedule == 0) {
+            // If we still have no schedule, return 0 supply royalty.
+            return 0;
+        }
+
+        totalRoyaltyMints = (mintAmount + (_totalMinted() % royaltyMintSchedule)) / (royaltyMintSchedule - 1);
+
+        if (totalRoyaltyMints > 0) {
+            address royaltyRecipient = config.fundsRecipient;
+
+            // If we have no recipient set, return 0 supply royalty.
+            if (royaltyRecipient == address(0)) {
+                return 0;
+            }
+            _mintNFTs(royaltyRecipient, totalRoyaltyMints);
```

Comment:
should this be returning something since every other path returns 0 ?

Reply:
the function returns `totalRoyaltyMints` which is set above

Reply:
Oh I see - totalRoyaltyMints is assigned above. It looks like the return value is not being used by the calling function.   How about not returning anything then out of this function since the return value is not used?  Would make the code more readable and save a bit on gas.